### PR TITLE
Ux 699 listboxbrowser access pass 1

### PR DESCRIPTION
**Pull request by tristanjasper**
### Purpose 🚀
This PR ensures listBoxBrowser correctly shows focus on a single list box item at any time.
Additionally changes the border style of the column when focused and fixes the focus style box being cut off.

Addresses issues 1,  8 and focus outline cutting off.
of 
https://aclgrc.atlassian.net/wiki/spaces/UX/pages/662410769/Listbox+Browser+Component+Review+11+22+2019

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [x ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/ux-699-listboxbrowser-acc-pass-1

### Screenshots 📸
Before:
accissues-before

After:
accissues

### References 🔗
https://aclgrc.atlassian.net/browse/UX-699

### CODE REVIEW

**Comment by tristanjasper** [comment]
I have closed https://github.com/acl-services/paprika/pull/344 and created this new PR to use correct branch number

File: packages/ListBox/src/components/Content/Content.styles.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const ContentStyled = styled.div`
+  :focus {
+    outline: 2px ${tokens.color.blackLighten30} dotted;
```
Comment (tristanjasper) [inline packages/ListBox/src/components/Content/Content.styles.js]: @jayde-acl Changed the dotted line colour to be as you requested

File: packages/ListBox/src/components/Content/Content.js [hunk 2]
```diff
@@ -41,25 +42,32 @@ const handleBlur = (state, dispatch) => () => {
...
 export default function Content(props) {
   const [state, dispatch] = useListBox();
   const { refListBoxContainer } = state;
 
   /* NOTE no idea what ROLE should be this div when the ListBox is INLINE */
-  /* eslint-disable jsx-a11y/no-static-element-interactions */
   if (state.isInline) {
     return (
-      <div
+      <ContentStyled
```
Comment (tristanjasper) [inline packages/ListBox/src/components/Content/Content.js]: Currently this logic of removing the focus style of the list items when focus is moved away from the popover content is only happening when the listbox component with prop `isInline`. I ran into some issues when the listBox was simply in its basic configuration (not inline) but perhaps could be revisited if required in future.

**Comment by nahumzs** [comment]
can you remove this http://storybooks.highbond-s3.com/paprika/ux-699-listboxbrowser-acc-pass-1/?path=/story/listbox-more-examples--columns-select story I don't think we need it now the listBoxBrowser is almost complete

**Comment by tristanjasper** [comment]
> can you remove this http://storybooks.highbond-s3.com/paprika/ux-699-listboxbrowser-acc-pass-1/?path=/story/listbox-more-examples--columns-select story I don't think we need it now the listBoxBrowser is almost complete

Ok removed!

File: packages/ListBox/src/components/Content/Content.styles.js (hunk 1 above)
Comment (mikrotron) [reply packages/ListBox/src/components/Content/Content.styles.js]: Coming soon:
https://github.com/acl-services/paprika/pull/372/files#diff-35881852d236126a5c4a48174b177f8aR88

File: packages/ListBox/src/components/Content/Content.js (hunk 2 above)
Comment (nahumzs) [reply packages/ListBox/src/components/Content/Content.js]: but is not affecting the current one correct?

Comment (tristanjasper) [reply packages/ListBox/src/components/Content/Content.js]: It is affecting the the listbox in the listbox browser (as both listboxes are inline there). Is that what you mean by current one?

File: packages/ListBox/src/components/Content/Content.styles.js (hunk 1 above)
Comment (tristanjasper) [reply packages/ListBox/src/components/Content/Content.styles.js]: Dotted line will look slightly different using that mixin/helper but we can switch it over when its available. 
image

**Pull request merged**